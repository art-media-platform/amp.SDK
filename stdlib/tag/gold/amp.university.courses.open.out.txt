                                    amp.spec.tag.v0.7000  . ✙ בְּרֵאשִׁ֖ית בָּרָ֣א אֱלֹהִ֑ים אֵ֥ת הַשָּׁמַ֖יִם וְאֵ֥ת הָאָֽרֶץ  ✙ .
82SQZRQWR7BVDXCW0E5DKTT4ST591HET9   amp.spec.tag.v0.7000.✙.בְּרֵאשִׁ֖ית.בָּרָ֣א.אֱלֹהִ֑ים.אֵ֥ת.הַשָּׁמַ֖יִם.וְאֵ֥ת.הָאָֽרֶץ.✙

                                    amp.university.open.courses
1DC1XKJ1RENFYQXNWMCTY3S4TCCJ7ZUZM   amp.university.open.courses

                                    amp.dev.archive.us-en
29ZU9HP9DHFG3J1YK8YWB86CCVDV0S6N9   amp.dev.archive.us-en


                                    amp.university.courses.open.arts.science.history.101.Once.upon.a.Time
4GH62HG39UFH99WFDFSPTM2SXHXB7NG7V   amp.university.courses.open.arts.science.history.101.once.upon.a.time
                                    	.L01.Rise of Physical Sciences, Engineering, Steampower, and Mechanical Engineering, feat. Peter Wang
741P0610D5R209UXHS8QFTP0D2407YSGC   L01.rise.of.physical.sciences.engineering.steampower.and.mechanical.engineering.feat.peter.wang
                                    	.L02.Planetary Motion and Numerical Computation & Modeling
49RT2U1S5UCK39E4VCRS5GQWBPYS0BUUK   L02.planetary.motion.and.numerical.computation.&.modeling
                                    	.L03.A New Science I: Background & History cryptography, code breaking, and war fighting, feat. Peter Wang
907GCCP32DXCMFJ1P1JW9Q8E2JCYM2GGP   L03.a.new.science.i.background.&.history.cryptography.code.breaking.and.war.fighting.feat.peter.wang
                                    	.L04.A New Science II: The Microcomputer Revolution, Arpanet, interpreters, numerical methods, .computer-driven cryptography; "Computer Science" arrives
6TEJPY7B0QFYXBR31094DD4UV24PSQ2VN   L04.a.new.science.II.the.microcomputer.revolution.arpanet.interpreters.numerical.methods.computer-driven.cryptography;."computer.science".arrives
                                    	.L05.A New Science III: Miniaturization of PC & mobile hardware, computerization of securities markets and exchanges, video games
7P85T9W9NYTPJ4MD651B8HEJN7HH2HH3X   L05.a.new.science.III.miniaturization.of.PC.&.mobile.hardware.computerization.of.securities.markets.and.exchanges.video.games
                                    	.L06.A New Science IV: Community-Centric Systems
3DR3HXQB5FS13KK3NPM8X5HCFNFDCCBPQ   L06.a.new.science.IV.community-centric.systems
                                    	.L07.Physics Unbound 1: Rise of General Relativity, Quantum Theory in a Newtonian world hungry for answers, feat. Sean Carroll, PhD, Theoretical Physicist, Caltech
9Q1E548WN4NR1NFWH237HS3JVF4J2QGEF   L07.physics.unbound.1.rise.of.general.relativity.quantum.theory.in.a.newtonian.world.hungry.for.answers.feat.sean.carroll.phd.theoretical.physicist.caltech
                                    	.L08.Physics Unbound 2: Black Holes, Origins of Life & The Physical Universe (Big Bang), Quantum Theory, and open mysteries in modern physics
9U8NT8FE8RQUN05Q903BU28TBYV5K985E   L08.physics.unbound.2.black.holes.origins.of.life.&.the.physical.universe.(big.bang).quantum.theory.and.open.mysteries.in.modern.physics



