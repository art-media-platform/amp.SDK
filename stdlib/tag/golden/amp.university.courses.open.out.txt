                               amp.spec.tag.v705  . ✙ בְּרֵאשִׁ֖ית בָּרָ֣א אֱלֹהִ֑ים אֵ֥ת הַשָּׁמַ֖יִם וְאֵ֥ת הָאָֽרֶץ  ✙ .
  5MUHRRC000DHZWZ746N7GB4DT2   amp.spec.tag.v705.✙.בְּרֵאשִׁ֖ית.בָּרָ֣א.אֱלֹהִ֑ים.אֵ֥ת.הַשָּׁמַ֖יִם.וְאֵ֥ת.הָאָֽרֶץ.✙

                               amp.university.open.courses
  2PSQUSVECVUMHCJSFMS43TQYTX   amp.university.open.courses

                               amp.dev.archive.us-en
  4TM6DCU43WR93W5Y7066GUEBRQ   amp.dev.archive.us-en


                               amp.university.courses.open.arts.science.history.101.Once.upon.a.Time {
  1DW5RC2RCKBH640W5EJT2090JG   amp.university.courses.open.arts.science.history.101.once.upon.a.time.{
                               	L01.Rise of Physical Sciences, Engineering, Steampower, and Mechanical Engineering, feat. Peter Wang
  3KHSQRN0Z7Q716VG4U8YRBMH2K   L01.rise.of.physical.sciences.engineering.steampower.and.mechanical.engineering.feat.peter.wang
                               	L02.Planetary Motion and Numerical Computation & Modeling
  5X16VFX8KRJ75XR04CKM5THSYJ   L02.planetary.motion.and.numerical.computation.&.modeling
                               	L03.A New Science 1: Introduction to cryptography, code breaking, and war fighting
  1GV53GKH5D6DX4YTY2TX8CH8QU   L03.a.new.science.1-introduction.to.cryptography.code.breaking.and.war.fighting
                               	L04.A New Science 2: The Microcomputer Revolution, Arpanet, interpreters, numerical methods, and rise of computer science
  55J4T89BHNDSVKJ09X17P9WRMD   L04.a.new.science.2-the.microcomputer.revolution.arpanet.interpreters.numerical.methods.and.rise.of.computer.science
                               	L05.A New Science 3: Miniaturization of PC & mobile hardware, computerization of securities markets and exchanges, video games
  1FHT04HS7FB45C76WQ11N4PKM5   L05.a.new.science.3-miniaturization.of.PC.&.mobile.hardware.computerization.of.securities.markets.and.exchanges.video.games
                               	L06.A New Science 4: Community-Centric systems
  2C8YWHV03RBWG2EZZQ90Q2SMZE   L06.a.new.science.4-community-centric.systems
                               	L07.Physics Unbound 1: Rise of General Relativity, Quantum Theory in a Newtonian world hungry for answers, feat. Sean Carroll, PhD, Theoretical Physicist, Caltech
  5CH83KJ125Z0KGZY32U5SGF8RG   L07.physics.unbound.1-rise.of.general.relativity.quantum.theory.in.a.newtonian.world.hungry.for.answers.feat.sean.carroll.phd.theoretical.physicist.caltech
                               	L08.Physics Unbound 2: Black Holes, Origins of Life & The Physical Universe (Big Bang), Quantum Theory, and open mysteries in modern physics
  15C1H6RSF8R1897U4EZZVWCB17   L08.physics.unbound.2-black.holes.origins.of.life.&.the.physical.universe.(big.bang).quantum.theory.and.open.mysteries.in.modern.physics
                               }
  4UVGE6U09VDVZK5CVP8VM762JW   }


